### PR TITLE
[1.13] Let ItemGroup provide it's own tabs image resource location

### PR DESCRIPTION
**Pull request by vrtl**
This adds a getTabsImage() to the ItemGroup class, which is used for rendering tabs and the scroll bar handle in the creative inventory gui. It can be overriden by custom ItemGroups so as to provide textures that match their background images. Without this, custom ItemGroup backgrounds clash with the default tabs texture, which can be seen in the before image below.

Examples were temporarily implemented in the Minecraft source code for screenshots, as I'm unable to get test mods running in the Forge environment.

**Example Code:**
group_tabs

**Before:**

**After**

### CODE REVIEW

File: patches/minecraft/net/minecraft/client/gui/inventory/GuiContainerCreative.java.patch [hunk 1]
```diff
@@ -1,6 +1,14 @@
 --- a/net/minecraft/client/gui/inventory/GuiContainerCreative.java
 +++ b/net/minecraft/client/gui/inventory/GuiContainerCreative.java
-@@ -53,6 +53,8 @@
+@@ -42,7 +42,6 @@
+ 
+ @OnlyIn(Dist.CLIENT)
+ public class GuiContainerCreative extends InventoryEffectRenderer {
+-   private static final ResourceLocation field_147061_u = new ResourceLocation("textures/gui/container/creative_inventory/tabs.png");
```
Comment (williewillus) [inline patches/minecraft/net/minecraft/client/gui/inventory/GuiContainerCreative.java.patch]: no need to remove this, just leave it there. saves the patches

File: patches/minecraft/net/minecraft/item/ItemGroup.java.patch [hunk 2]
```diff
@@ -9,7 +9,15 @@
     public static final ItemGroup field_78030_b = (new ItemGroup(0, "buildingBlocks") {
        @OnlyIn(Dist.CLIENT)
        public ItemStack func_78016_d() {
-@@ -104,7 +104,16 @@
+@@ -95,6 +95,7 @@
+          return new ItemStack(Blocks.field_150486_ae);
+       }
+    }).func_78025_a("inventory.png").func_78022_j().func_78014_h();
++   private static final net.minecraft.util.ResourceLocation CREATIVE_INVENTORY_TABS = new net.minecraft.util.ResourceLocation("textures/gui/container/creative_inventory/tabs.png");
```
Comment (williewillus) [inline patches/minecraft/net/minecraft/item/ItemGroup.java.patch]: put patched-in fields near the methods they're used (put it above getTabsImage in this case)

**Comment by LexManos** [comment]
Also needs a better explanation, perhaps a test mod and example screenshots.

**Comment by vrtl** [comment]
This PR is also updated and working fine, just noticed the update tag was removed on my other PR but not this one.

**Pull request merged**